### DISCUSSION TITLE
Support Svelte - Sapper

### DISCUSSION THREAD

**Discussion by IRediTOTO** [tailwindlabs/tailwindcss#3729] [Ideas]
Hi, I just found Svelte :) It's really cool, I'm trying to config Tailwind with Svelte/Sapper. Hope you can add it to the document . 
Like this post to get Attention

**Comment by NovusOpus** [comment]
There's some examples of Setting Up tailwind with Svelte/Sapper here: 

**Svelte**: https://github.com/tailwindlabs/tailwindcss-setup-examples/tree/master/examples/svelte
**Sapper**: https://github.com/tailwindlabs/tailwindcss-setup-examples/tree/master/examples/sapper

There's also a template you can clone/use although also includes TypeScript, ESLint, and Prettier : https://github.com/tonyketcham/sapper-tailwind2-template

But also hope they can add a final/official guide into the docs soon.
